src/components/AccountDrawer/index.tsx

```diff
@@ -18,7 +18,7 @@ const DRAWER_MARGIN = '8px'
 const DRAWER_OFFSET = '10px'
 const DRAWER_TOP_MARGIN_MOBILE_WEB = '72px'
 
-const accountDrawerOpenAtom = atom(false)
+export const accountDrawerOpenAtom = atom(false)
```

Comment:
I'd rather this not be exported, because it's safer to wrap it in helpers. In this case, you can use `const [accountDrawerOpen] = useAccountDrawer()` for the same functionality.

Reply:
This would be better if the landing page content faded out. Opacity change easeInOut at the same speed of the drawer animation. I think it might be 250ms